src/EntityMetaRelationInstaller.php

```diff
@@ -104,4 +98,27 @@ public function installEntityMetaTypeOnContentEntityType(string $entity_meta_typ
 
   }
 
+  /**
+   * Update target bundles in requested field configs.
+   *
+   * @param string $emr_bundle
+   *   The entity meta relation bundle.
+   * @param string $field_name
+   *   The field name.
+   * @param array $target_bundles
+   *   The list of allowed bundles.
+   */
+  protected function updateTargetBundlesInFields(string $emr_bundle, string $field_name, array $target_bundles): void {
+    /** @var \Drupal\Core\Field\FieldConfigInterface $field_config */
+    $field_config = $this->entityTypeManager->getStorage('field_config')->load("entity_meta_relation.{$emr_bundle}.{$field_name}");
+    if (!$field_config) {
+      return;
```

Comment:
Let's throw an exception here. Before it used to fail somehow if the field was not found so we could catch if the thing doesnt get installed properly.